core/CMakeLists.txt

```diff
@@ -11,7 +11,24 @@ else()
     set(library_type "SHARED")
 endif()
 
-if(IOS OR MSVC OR APPLE)
+if(IOS OR APPLE)
+    # We need tinyxml2 for the camera definition parsing.
+    add_subdirectory(third_party/tinyxml2 EXCLUDE_FROM_ALL)
```

Comment:
Seems like this is done for all configs, so I'd move it in from of this `if` block. And if you know why we don't `include_directories(SYSTEM third_party/tinyxml2)` for Android, I'd be interested to know :-).

Reply:
You're right, and omitting the `include_directories` for Android is an oversight, thx. I'll fix it.